pkg/statistics/fmsketch.go

```diff
@@ -42,10 +42,22 @@ var fmSketchPool = sync.Pool{
 	},
 }
 
-// FMSketch is used to count the number of distinct elements in a set.
+// TODO: add this attribute to PB and persist it instead of using a fixed number(executor.maxSketchSize)
+// MaxSketchSize is the maximum size of the hashset in the FM sketch.
+const MaxSketchSize = 10000
```

Comment:
```suggestion
// MaxSketchSize is the maximum size of the hashset in the FM sketch.
// TODO: add this attribute to PB and persist it instead of using a fixed number(executor.maxSketchSize)
const MaxSketchSize = 10000
```